effekt/shared/src/main/scala/effekt/generator/js/Transformer.scala

```diff
@@ -53,7 +53,7 @@ trait Transformer {
 
     val other = freshName("other")
     def otherGet(field: JSName): js.Expr = js.Member(js.Variable(other), field)
-    def compare(field: JSName): js.Expr = js"${get(field)} !== ${otherGet(field)}"
+    def compare(field: JSName): js.Expr = js"!${$effekt.call("equals", get(field), otherGet(field))}"
```

Comment:
This seems a bit low-level outside of the JS AST representation, but there's probably a reason for it, right?
Otherwise it would be something like `!${js.MethodCall(js"$effekt", js"equals", get(field), otherGet(field))}`?

Reply:
Ah, it's Scala code! `$effekt.call` is a method in `Tree.scala`.
Nevermind, retracting.